src/WebJobs.Script/ExtensionRequirements/ExtensionStartupTypeRequirement.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Azure.WebJobs.Script.ExtensionRequirements
 {
-    internal sealed class ExtensionStartupTypeRequirement
+    public sealed class ExtensionStartupTypeRequirement
```

Comment:
Curious why this and `BundleRequirement` need to be made public?

Reply:
I needed to make it public because the type is used in ExtensionRequirementOptions. Inorder for options to hydrate from IConfiguration, its type needs to be public.